src/sentry/relocation/tasks/process.py

```diff
@@ -320,6 +329,10 @@ def uploading_start(uuid: UUID, replying_region_name: str | None, org_slug: str
     # 10 minutes per try.
     soft_time_limit=60 * 10,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
+        retry=Retry(times=4, on=(Exception,), times_exceeded=LastAction.Discard),
```

Comment:
Should this task have a processing_deadline of 10 minutes to align with the `soft_time_limit` option?